src/docfx/cli/Docfx.cs

```diff
@@ -84,6 +85,16 @@ private static Command RestoreCommand()
             return command;
         }
 
+        private static Command ApplyTemplateCommand()
+        {
+            var command = CreateCommand("apply-template", "Apply templates", ApplyTemplate.Run);
+            DefineCommonCommands(command);
+            command.AddOption(new Option<string>("--locale", "Locale of the files."));
+            command.AddOption(new Option<string>("--input", "Input directory of structured json files."));
+            command.AddOption(new Option<string>("--output", "Output directory path."));
```

Comment:
Make this line the same as other output options. Use `-o, --output` and change the summary to match others.